### PR TITLE
Balance Component setup

### PR DESCRIPTION
**Pull request by icerove**
Create a new Component named Balance and it shows the amount more flex and has near and millinear token image

### CODE REVIEW

File: src/components/Balance/Balance.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    let amount = props.amount / BALANCE
+    let style = {
+        width: "1.2em",
+        height: "1.2em",
+        marginLeft: "4px", 
+        bottom: "-2px",
+        position: "relative"
+    }
+    return (<div>
+        {(1234 < 0.01) ?
```
Comment (ilblackdragon) [inline src/components/Balance/Balance.js]: This should be amount, right?

File: src/components/Balance/Balance.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+            <div>{amount.toFixed(4)}<img style={style} src={props.milli} alt="" /></div> : <div>{amount} Ⓝ</div>}
```
Comment (ilblackdragon) [inline src/components/Balance/Balance.js]: And if it's in millis - it should be * 1000

File: src/components/Balance/Balance.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+const BALANCE = 1000000000000000000
```
Comment (ilblackdragon) [inline src/components/Balance/Balance.js]: It's not really a balance, but a NEAR nomination
Also better to write 10**18 and a comment that this is denomination of one near in minimal non divisible units (attoNears)

File: src/components/dashboard/DashboardContainer.js [hunk 4]
```diff
@@ -2,7 +2,9 @@ import React from 'react'
 import { Link } from 'react-router-dom'
 
 import DashboardOtherAssets from './DashboardOtherAssets'
+import Balance from '../Balance/Balance'
 
+import milli from '../../images/n-1000.svg'
```
Comment (ilblackdragon) [inline src/components/dashboard/DashboardContainer.js]: I don know but is there a reasob not to import image once in Balance component?

File: src/components/profile/Profile.js [hunk 5]
```diff
@@ -32,9 +32,11 @@ class Profile extends Component {
             loader: false
          }))
       }, 1000)
+
```
Comment (ilblackdragon) [inline src/components/profile/Profile.js]: These new lines are required?

File: src/components/Balance/Balance.js (hunk 1 above)
Comment (icerove) [reply src/components/Balance/Balance.js]: yes, I just test with 1234

File: src/components/dashboard/DashboardContainer.js (hunk 4 above)
Comment (icerove) [reply src/components/dashboard/DashboardContainer.js]: it has black one and write one, this is easier to understand as giving as props

File: src/components/profile/Profile.js (hunk 5 above)
Comment (icerove) [reply src/components/profile/Profile.js]: I used to console log the props, now it is no more used

File: src/components/Balance/Balance.js (hunk 3 above)
Comment (icerove) [reply src/components/Balance/Balance.js]: right, added

File: src/components/Balance/Balance.js (hunk 1 above)
Comment (ilblackdragon) [reply src/components/Balance/Balance.js]: just a tip: it's easier to test components, by passing props vs changing code in the component

Comment (icerove) [reply src/components/Balance/Balance.js]: thank you

**Pull request merged**